server/embed/etcd_test.go

```diff
@@ -0,0 +1,24 @@
+package embed
```

Comment:
The license header is missing in this new file!

Reply:
Why is header validation passing?